bigtable/tests/unit/test_instance.py

```diff
@@ -897,6 +897,123 @@ def test_list_app_profiles(self):
         self.assertIsInstance(app_profile_2, AppProfile)
         self.assertEqual(app_profile_2.name, app_profile_name2)
 
+    def test_get_iam_policy(self):
+        from google.cloud.bigtable_admin_v2.gapic import (
+            bigtable_instance_admin_client)
+        from google.iam.v1 import iam_policy_pb2
+        from google.iam.v1 import policy_pb2
+        from google.cloud.iam import Policy
+
+        credentials = _make_credentials()
+        client = self._make_client(project=self.PROJECT,
+                                   credentials=credentials, admin=True)
+        instance = self._make_one(self.INSTANCE_ID, client)
+
+        version = 1
+        etag = b'etag_v1'
+        bindings = [{'role': 'roles/owner',
+                     'members': ['serviceAccount:service_acc1@test.com',
+                                 'user:user1@test.com']}]
+
+        expected_request_policy = policy_pb2.Policy(version=version,
+                                                    etag=etag,
+                                                    bindings=bindings)
+
+        expected_request = iam_policy_pb2.GetIamPolicyRequest(
+            resource=instance.name
+        )
+
+        # Patch the stub used by the API method.
+        channel = ChannelStub(responses=[expected_request_policy])
+        instance_api = (
+            bigtable_instance_admin_client.BigtableInstanceAdminClient(
+                channel=channel))
+        client._instance_admin_client = instance_api
+        # Perform the method and check the result.
+        policy_request = Policy(etag=etag, version=version)
+        policy_request.owners = [Policy.user("user1@test.com"),
+                                 Policy.service_account(
+                                     "service_acc1@test.com")]
+
+        result = instance.get_iam_policy()
+        actual_request = channel.requests[0][1]
+
+        self.assertEqual(actual_request, expected_request)
+        self.assertEqual(result.owners, policy_request.owners)
+
+    def test_set_iam_policy(self):
+        from google.cloud.bigtable_admin_v2.gapic import (
+            bigtable_instance_admin_client)
+        from google.iam.v1 import iam_policy_pb2
+        from google.iam.v1 import policy_pb2
+        from google.cloud.iam import Policy
+
+        credentials = _make_credentials()
+        client = self._make_client(project=self.PROJECT,
+                                   credentials=credentials, admin=True)
+        instance = self._make_one(self.INSTANCE_ID, client)
+
+        version = 1
+        etag = b'etag_v1'
+        bindings = [{'role': 'roles/owner',
```

Comment:
Please use one of the roles supported by Bigtable `Instance` resource.
* `'roles/bigtable.admin'`
* `'roles/bigtable.user'`
* `'roles/bigtable.reader'`
* `'roles/bigtable.viewer'`

Reply:
Done, thanks.